_includes/javascript-utils.html

```diff
@@ -0,0 +1,5 @@
+<script>
+    $(document).ready(function(){
+    $('table').addClass('table is-dark');
+    });
+</script>
```

Comment:
This is smelly, can we just change the default styling for table?

Reply:
Fixed